hashtagsv2/hashtags/forms.py

```diff
@@ -1,4 +1,5 @@
 from django import forms
+TYPE_CHOICES = [('or', 'or'), ('and', 'and')]
```

Comment:
Minor code style improvement - this should be separated from the imports above (I think one newline is fine).